src/utils/scrap_data.ts

```diff
@@ -0,0 +1,176 @@
+import puppeteer, { Browser, Page } from 'puppeteer';
+import { VIVINO_GRAPES_API, VIVINO_SEARCH_URL, VIVINO_EXPLORE_API, WINES_PER_PAGE, VIVINO_WINE_STYLES_API, VIVINO_FOODS_API } from '../const/const';
+import { REGIONS } from '../const/regions';
+import { WINE_TYPES } from '../const/wine_types';
+import { WineParams } from '../interfaces/params.interface';
+import { EXPLORE_API_PARAMS } from '../enums/explore_api_params.enum';
+import { Grape, Wine, WineStyle, Food } from '../interfaces/wine.interface';
+
+class ScrapData {
+  grapes: Grape[] = [];
+  wineStyles: WineStyle[] = [];
+  foods: Food[];
+
+  appendParam = (param: string, queryKey: string, list: any = []) => {
```

Comment:
Could this lib https://www.npmjs.com/package/qs can help to remove the custom code?

Reply:
This is very helpful package. Yes, custom code can be removed if I use this package.

Reply:
Yes another goodie from TJ Holowaychuk!